<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>USBJrkDialog</class>
 <widget class="QDialog" name="USBJrkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>USB Jrk Motor Contol</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>606</width>
     <height>448</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="mainLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetFixedSize</enum>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QComboBox" name="devicesCB">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="refreshBtn">
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="minimumSize">
       <size>
        <width>581</width>
        <height>400</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Feedback</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>556</width>
          <height>338</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QCheckBox" name="feedbackInvertedCb">
           <property name="text">
            <string>Inverted</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Minimum</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Maximum</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Feedback</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="feedbackMin">
           <property name="maximum">
            <number>4095</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="feedbackMax">
           <property name="maximum">
            <number>4095</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Disconnect</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="feedbackDisconnectMin">
           <property name="maximum">
            <number>4095</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="feedbackDisconnectMax">
           <property name="maximum">
            <number>4095</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Set Target</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="3">
          <widget class="QSlider" name="targetSlider">
           <property name="minimumSize">
            <size>
             <width>321</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximum">
            <number>4095</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBothSides</enum>
           </property>
           <property name="tickInterval">
            <number>128</number>
           </property>
          </widget>
         </item>
         <item row="7" column="3">
          <widget class="QLabel" name="targetLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Feedback</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="feedbackLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="4">
          <widget class="QPushButton" name="stopBtn">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Scaled</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLabel" name="scaledfbLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Degrees</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QLabel" name="degreesLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="10" column="2">
          <widget class="QLabel" name="scaleddegreesLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="12" column="2">
          <widget class="QDoubleSpinBox" name="gotoDegSb">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Sensor</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Track</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Accuracy</string>
           </property>
          </widget>
         </item>
         <item row="13" column="2">
          <widget class="QComboBox" name="accCb">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>0.1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>0.01</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>0.001</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>0.0001</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="11" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>100</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="13" column="3">
          <widget class="QPushButton" name="gotodegBtn">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="10" column="3">
          <widget class="QDoubleSpinBox" name="degSb">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Calibrate</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="calibrateMinFeedbackBtn">
           <property name="text">
            <string>Read</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="calibrateMaxFeedbackBtn">
           <property name="text">
            <string>Read</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QPushButton" name="applyFeedbackBtn">
           <property name="text">
            <string>Write to Jrk</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Motor</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Status</string>
       </attribute>
       <widget class="QPushButton" name="clrerrorsBtn">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>10</y>
          <width>98</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Clear errors</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Degrees Calibration</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>421</width>
          <height>151</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Minimum</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Maximum</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Feedback</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="feedbackMinDegSb">
           <property name="maximum">
            <number>4095</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="feedbackMaxDegSb">
           <property name="maximum">
            <number>4095</number>
           </property>
           <property name="value">
            <number>4095</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Degrees</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="mindegSb">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QDoubleSpinBox" name="maxdegSb">
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="value">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QPushButton" name="applyDegBtn">
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="gotoMinBtn">
           <property name="text">
            <string>Run to min</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPushButton" name="gotoMaxBtn">
           <property name="text">
            <string>Run to max</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Record LUT</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Scaled</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="scaledfeedbackMinDegSb">
           <property name="maximum">
            <number>4095</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="scaledfeedbackMaxDegSb">
           <property name="maximum">
            <number>4095</number>
           </property>
           <property name="value">
            <number>4095</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
